Containers/ibm_fncm_cr_enterprise_FC_content_CMOD_CM8_Users.txt

```diff
@@ -0,0 +1,394 @@
+  #   collectd_plugin_write_graphite_port: 2003
+  #   collectd_interval: 10
+  #   collectd_disable_host_monitoring: false
+  #   collectd_plugin_write_prometheus_port: 9103
+
+  # # Logging setting
+  # logging_configuration:
+  #   mon_log_parse: false
+  #   mon_log_service_endpoint: localhost:5044
+  #   private_logging_enabled: false
+  #   logging_type: default
+  #   mon_log_path: /path_to_extra_log
+
+  ########################################################################
+  ########   IBM Business Automation Navigator configuration      ########
+  ########################################################################
+  navigator_configuration:
+    ## Navigator secret that contains user credentials for LDAP and database
+    ban_secret_name: ibm-ban-secret
+
+    ## By default all the components create ingress and routes with required annotations. In case any custom annotation is needed for the environment provide below.
+    #    route_ingress_annotations:
+    #      - haproxy.router.openshift.io/balance: roundrobin
+    route_ingress_annotations:
+
+    ## The architecture of the cluster.  This is the default for Linux and should not be changed.
+    arch:
+      amd64: "3 - Most preferred"
+
+    ## The number of replicas or pods to be deployed.  The default is 1 replica and for high availability in a production env,
+    ## it is recommended to have 2 or more.
+    replica_count: 2
+
+    ## This is the image repository and tag that correspond to image registry, which is where the image will be pulled.
+#    image:
+#      ## The default repository is the IBM Entitled Registry
+#      repository: cp.icr.io/cp/cp4a/ban/navigator
+#      tag: ga-309-icn
+#
+#      ## This will override the image pull policy in the shared_configuration.
+#      pull_policy: IfNotPresent
+
+     image:
+       ## The default repository is the IBM Entitled Registry
+      repository: icr.io/cpoen/icn/navigator
+      tag: ga-3012-icn
+
+       ## This will override the image pull policy in the shared_configuration.
+      pull_policy: IfNotPresent
+
+    ## Logging for workloads.  This is the default setting.
+    log:
+      format: json
+
+    ## The initial resources (CPU, memory) requests and limits.  If more resources are needed,
+    ## make the changes here to meet your requirement.
+    resources:
+      requests:
+        cpu: 500m
+        memory: 512Mi
+      limits:
+        cpu: 1
+        memory: 1536Mi
+
+    ## By default "Autoscaling" is disabled with the following settings with a minimum of 2 replca and a maximum of 3 replicas.  Change
+    ## this settings to meet your requirement.
+    auto_scaling:
+      enabled: false
+      max_replicas: 3
+      min_replicas: 2
+      ## This is the default cpu percentage before autoscaling occurs.
+      target_cpu_utilization_percentage: 80
+
+#    ## send email
+#    java_mail:
+#      host: "fncm-exchange1.ibm.com"
+#      port: "25"
+#      sender: "MailAdmin@fncmexchange.com"
```

Comment:
the "host" and "sender" information should be removed from this file.

Reply:
Corrected